src/Definition/Entity.php

```diff
@@ -89,6 +89,9 @@ public function setClass(string $class): self
         return $this;
     }
 
+    /**
+     * @return class-string|null
```

Comment:
You can add class template like TEntity and use it here and with other interfaces like
`class-string<RepositoryInterface<TEntity>>|null`?

Reply:
Added template